tests/integration_tests/performance/test_network_ab.py

```diff
@@ -67,7 +66,8 @@ def network_microvm(request, microvm_factory, guest_kernel, rootfs):
 
 @pytest.mark.nonci
 @pytest.mark.parametrize("network_microvm", [1], indirect=True)
-def test_network_latency(network_microvm, metrics):
+@pytest.mark.parametrize("iteration", [1, 2])
```

Comment:
May want to copy the explanation in the commit to the top of this file so that we don't switch it back to 1 iteration with 900 pings.

Reply:
Done :)